### ISSUE TITLE
Question: loss value is a little large.

### ISSUE THREAD

**Issue by asimay** [Randl/MobileNetV2-pytorch#8]
Hi, dear:
From here:
https://github.com/Randl/MobileNetV2-pytorch/blob/master/results/mobilenet_v2_0.5_160/results.csv
I can see the loss value is still a little large for  a good module, from loss definition, loss should be between 0-1, and when nearby 0, it is more accurate, but from the excel the loss is still above 1.
Could you please explain this issue? or any optimization need to be done?
 Thanks!

**Comment by Randl** [comment]
Why loss should be between 0-1?

**Comment by asimay** [comment]
sorry. as I know, many loss functions are defined with cross-entropy loss function. so I think it is between 0-1. Am I right?

**Comment by Randl** [comment]
cross entropy is not bounded, thus no, you're not right
